rps_bench_profile.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+VM_NUM=$1
+
+if [ $VM_NUM -eq 1 ]
+then
+DELAY=35000
+RPS=5
+elif [ $VM_NUM -eq 10 ]
+then
+DELAY=140900
+RPS=50
+else
+echo -e "VM_NUM argument invalid\n"
+exit
+fi
+
+sudo mkdir -m777 -p /tmp/ctrd-logs && sudo env "PATH=$PATH" /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 1>/tmp/ctrd-logs/fccd_orch_noupf_log_bench.out 2>/tmp/ctrd-logs/fccd_orch_noupf_log_bench.err &
+
+perf stat -D $DELAY -a -e instructions,LLC-loads,LLC-load-misses,LLC-stores,LLC-store-misses --output perf-${VM_NUM}VMs-lr_training.profile sudo env "PATH=$PATH" go test -v -race -cover -run TestBenchRequestPerSecond -args -vm $VM_NUM -requestPerSec $RPS -executionTime 2
```

Comment:
you should be more careful when profiling. Set a conservative delay to guarantee that all VMs have booted and start executing. Make sure that perf profiles only the middle of execution -- not the beginning or the end of the benchmark, which include allocation, deallocation of resources by Go (or even compilation). Hence, execution time should be longer.

I suggest to attach perf to the go process dynamically to profile the middle of the execution, avoiding the ends.